pkg/issuercerts/issuercerts.go

```diff
@@ -125,7 +125,7 @@ func CheckExpiringSoon(cert *x509.Certificate) error {
 // we have posed on the public key and signature algorithms
 func CheckCertAlgoRequirements(cert *x509.Certificate) error {
 	if cert.PublicKeyAlgorithm == x509.ECDSA {
-		// this si a safe cast here as wel know we are using ECDSA
+		// this si a safe cast here as well know we are using ECDSA
```

Comment:
`is` is misspelled. I can fix that as well. I'm not sure offhand why I missed that.

Reply:
reading it again, I think this change is wrong and that the correct word is `we` not `well`.